examples/python-dataflow/dataflow_without_webcam.yml

```diff
@@ -0,0 +1,27 @@
+communication:
+  zenoh:
+    prefix: /example-python-dataflow
```

Comment:
We should use a different prefix here to avoid collisions when this dataflow is run in parallel with `dataflow.yml`.

Reply:
I will add a prefix for the `no-webcam` but the webcam logic is not going to be tested in the CI, for obvious reasons, but I still agree that another prefix is still a good idea.`